src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/instantiate/containers/PartySelection.tsx

```diff
@@ -278,7 +278,7 @@ const PartySelectionWithRouter = withRouter((props: IPartySelectionProps) => {
   }
 
   return (
-    <InstantiationContainer>
+    <InstantiationContainer type='partyChoice'>
```

Comment:
would this be a good point in time to consider renaming the type partySelection to match the file name?